README.md

```diff
@@ -55,33 +61,59 @@ After you have installed GOV.UK Frontend the `@govuk-frontend` package will appe
 
 ## Import assets
 
-Import styles into your main Sass file.
+### Import styles
+
+You need to import the GOV.UK Frontend styles into the main Sass file in your project. You should place the below code before your own Sass rules (or Sass imports) if you want to override GOV.UK Frontend with your own styles.
 
-To import all components, add the line:
+1. To import all components, add the below to your Sass file:
 ```CSS
 @import "@govuk-frontend/all/all";
 ```
 
-To import an individual component (for example, a button), add the line:
+2. To import an individual component (for example a button), add the below to your Sass file:
 ```CSS
 @import "@govuk-frontend/button/button";
 ```
 
+In order to resolve the above `@import` paths(s) in your build, you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) (in Ruby, they should be added to [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths)).
 
-To resolve your `@import` declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) in Ruby.
+For example, if your project uses Gulp, you would add the Sass include paths to your Gulp configuration file (for example `gulpfile.js`) with [gulp-sass](https://www.npmjs.com/package/gulp-sass). Below is an example:
 
-Below is a code sample you can add to your gulp configuration file using the gulp-sass package:
 ```JS
-.pipe(sass({ includePaths: 'node_modules/' }))
+gulp.task('sass', function () {
+  return gulp.src('./sass/**/*.scss')
+    .pipe(sass({
+      includePaths: 'node_modules/@govuk-frontend'
```

Comment:
I'm not sure this includePath will work since the imports above reference @govuk-frontend.

Non-blocking suggestion: We could add 'node_modules' prefix to the imports above, and change this section to 'Don't want to include node_modules? Do this'

Reply:
Thanks @nickcolley I've fixed this.

Reply:
This works with the example below. I was undecided whether to include `@govuk-frontend` in `includePaths` or in the Sass import. But I've switched these around now, that does make more sense.